### ISSUE TITLE
Create a procedure for storing user infomation into the database

### ISSUE THREAD

**Issue by fhuang566** [cse442-at-ub/project_s23-one-belt-one-road#44]
**Task tests**

*Test 1*
1) Go to database management page; link: https://www-student.cse.buffalo.edu/tools/db/phpmyadmin/db_structure.php?server=1&db=cse442_2023_spring_team_j_db
2) Open "Procedures" tab
3) Verify that there exists a procedure called "createUser"
4) Execute the procedure with specified parameters including "username", "password", and "email"
5) Go to "user" table and make certain that the latest record match all the infomation added.

**Comment by fhuang566** [comment]
**Task tests**

*Test 1*
1) Go to database management page; link: https://www-student.cse.buffalo.edu/tools/db/phpmyadmin/db_structure.php?server=1&db=cse442_2023_spring_team_j_db
2) Open "Procedures" tab
3) Verify that there exists a procedure called "createUser"
4) Execute the procedure with specified parameters including "username", "password", and "email"
5) Go to "user" table and make certain that the latest record match all the infomation added.

**Comment by anht-11** [comment]
Task should not be blocked by another task. You might consider this task block the user story #35 instead
